.github/workflows/validate-submission.yaml

```
@@ -38,7 +53,7 @@ jobs:
           v <- hubValidations::validate_pr(
               gh_repo = Sys.getenv("GITHUB_REPOSITORY"),
               pr_number = Sys.getenv("PR_NUMBER"),
-              skip_submit_window_check = FALSE
+              skip_submit_window_check = if (Sys.getenv("model") == "ensemble") TRUE else FALSE
```

It's not necessary to have a whole if/else thing here. This expression will evaluate to TRUE or FALSE

```suggestion
              skip_submit_window_check = Sys.getenv("model") == "ensemble"
```